golang/cmd/encrypt/README.md

```diff
@@ -11,21 +11,14 @@ document is outputted to the output_file path given as a flag.
 ## Installation:
 
 ```shell
-# Clone repo
-git clone https://github.com/subscriptions-project/encryption
-
-# Open repo directory
-cd encryption/
-
-# Install deps
-go get -d ./...
+# Go get the script
+go get github.com/subscriptions-project/encryption/golang/cmd/encrypt
```

Comment:
Add -u flag to go get command.

Reply:
Also this doesn't appear to work properly. I get an error (I tried on OSX and Goobuntu). I think because it can't find the tink dependency.  Consider storing tink in a vendor dir or use go modules.

```
$ go get github.com/subscriptions-project/encryption/golang/cmd/encrypt
unexpected directory layout:
	import path: _/Users/angielin/go/src/github.com/subscriptions-project/encryption/golang/pkg/encryption
	root: /Users/angielin/go/src
	dir: /Users/angielin/go/src/github.com/subscriptions-project/encryption/golang/pkg/encryption
	expand root: /Users/angielin/go
	expand dir: /Users/angielin/go/src/github.com/subscriptions-project/encryption/golang/pkg/encryption
	separator: /
```

Reply:
Added Tink inside of vendor. Hopefully that solves things!